src/util.jl

```diff
@@ -27,3 +27,86 @@ function rmerge(def_s::Associative, add_s::Associative)
     end
     return resp
 end
+
+############################
+# norms for list of points #
+############################
+
+function absmax(s::ListOfPoints)
+    t = 0.0
+    for p in s
+        t = max(t, maximum(p))
+    end
+    t
+end
+absmax(x::Vector{<:ListOfPoints}) = maximum(absmax.(x))
+
+import Base
+function Base.maxabs(v::ListOfPoints{d}) where d
```

Comment:
This is _technically_ type piracy (we don't own the function or the type), but I'll let it slide :)

Reply:
and `maxabs` is deprecated. But I don't know that the syntax `maximum(abs, ...)` would apply here.